### DISCUSSION TITLE
Use viem with ethereum signer (withpaper)

### DISCUSSION THREAD

**Discussion by MoMannn** [wevm/viem#970] [Question]
I am integrating withpaper into my project which returns an ethers signer object for the authenticated user. How can I use or transform that ethers signer to user it in viem?

**Comment by tmm** [comment]
You can create a custom account: https://viem.sh/docs/accounts/custom.html. Here's an [example implementation](https://viem.sh/docs/migration-guide.html#removed-etherswallettoaccount.).
